### DISCUSSION TITLE
How can we set chart datetime language?

### DISCUSSION THREAD

**Discussion by andrelucassc** [apache/superset#18260] [Q&A / Help]
I've followed guidelines from  [Issue 3068](https://github.com/apache/superset/issues/3068) to change the language to "pt_BR" and the UI in most parts has taken the expected change, but there is one major area that this was not the case. 

Creating a linechart with a metric and a datetime component results in the date beeing in english. The date rendered in the visualizations (example Oct 15 2021) can be altered? Is it something that has to be taken into account what is used to build those visualizations in the first place?

**Comment by andrelucassc** [comment]
Example of what I mean:

![problem date chart6bc2aa76a67d6fb7192f11d9ec1cc1129191f48fa546fcab2d79f27a43b57047](https://user-images.githubusercontent.com/20599467/105501648-3e534000-5ca3-11eb-85cc-edd080b277b8.png)

Expected Outcome:
Fev 2012

**Comment by junlincc** [comment]
@andrelucassc Thanks for reporting, André. what version are you at?

**Comment by andrelucassc** [comment]
Hi, @junlincc , the version was 0.38.0.
